src/extras/relayPagination.ts

```diff
@@ -166,25 +166,18 @@ export const relayPagination = (params: PaginationParams = {}): Resolver => {
       const page = getPage(cache, linkKey);
       if (page === null) {
         continue;
-      } else if (
+      }
+
+      if (
         mergeMode === 'inwards' &&
         typeof args.last === 'number' &&
         typeof args.first === 'number'
       ) {
-        // This is a special but rare case for simultaneously merging first and last edges
-        if (page.edges.length < args.first + args.last) {
-          startEdges = concatEdges(cache, startEdges, page.edges);
-          pageInfo.hasNextPage = false;
-          pageInfo.hasPreviousPage = false;
-          pageInfo.startCursor = null;
-          pageInfo.endCursor = null;
-          break;
```

Comment:
I'm not familiar with what's the purpose of this condition, but I could remove it without making any existing test fail, and it allows the added tests to pass.

I found that when e.g the GraphQL backend returns less edges than the ones requested, this condition was triggered causing the `edges` object to be `null`.

If I'm missing something and there's a better way to rewrite this condition without removing it lmk!

Reply:
Yea, seems safe to remove them :)